### PR TITLE
cell death curve figure - supplemental

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adds a cell death curve figure from data collected from Carla's lab.

### CODE REVIEW

File: figures/death_curves/figures/death_curve.png
Comment (jenna-tomkinson) [inline figures/death_curves/figures/death_curve.png:1]: I like the similar colors based on the treatment! I don't have any comments on this figure, just one overall question. 

Is this result expected where we see a similar percentage of dead cells from 3 to 6 hours? I would have expected a slightly large percentage from 3 to 6 but maybe we are seeing that pyroptosis takes longer to get to the final "dead" stage.

File: figures/death_curves/scripts/deathcurves_plot.r [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+           y = "Percentage of dead cells")
+    + facet_grid(~time)
+    + theme(
+        axis.text.x = element_text(size = 18, angle = 45, hjust = 1),
+        axis.text.y = element_text(size = 18),
+        axis.title = element_text(size = 20),
+        strip.text = element_text(size = 20),
+        legend.text = element_text(size = 18),
+        legend.title = element_text(size = 20),
+        # legend.position = "none"
```
Comment (jenna-tomkinson) [inline figures/death_curves/scripts/deathcurves_plot.r:79]: Recommend dropping commented-out code.

**Review by jenna-tomkinson — APPROVED** [review]
Code and figure look clean! LGTM! :tada: Feel free to merge after addressing my comments. Glad to finally see these results!

File: figures/death_curves/scripts/deathcurves_plot.r (hunk 1 above)
Comment (MikeLippincott) [reply figures/death_curves/scripts/deathcurves_plot.r:79]: dropped

**Pull request merged**